docs/src/main/sphinx/security/tls.rst

```diff
@@ -12,17 +12,67 @@ This topic describes how to configure your Trino server to use :ref:`TLS
 All authentication technologies supported by Trino require configuring TLS as
 the foundational layer.
 
-When configured to use TLS, a Trino server responds to client connections using
-TLS 1.2 and TLS 1.3 certificates. The server rejects TLS 1.1, TLS 1.0, and all
-SSL format certificates.
-
 .. important::
 
     This page discusses only how to prepare the Trino server for secure client
     connections from outside of the Trino cluster to its coordinator.
 
 See the :doc:`Glossary </appendix/glossary>` to clarify unfamiliar terms.
 
+.. _tls-version-and-ciphers:
+
+Supported standards
+-------------------
+
+When configured to use TLS, the Trino server responds to client connections
+using TLS 1.2 and TLS 1.3 certificates. The server rejects TLS 1.1, TLS 1.0, and
+all SSL format certificates.
+
+The Trino server does not specify a set of supported ciphers, instead deferring
+to the defaults set by the JVM version in use. The documentation for Java 11
+lists its `supported cipher suites
+<https://docs.oracle.com/en/java/javase/11/security/oracle-providers.html#GUID-7093246A-31A3-4304-AC5F-5FB6400405E2__SUNJSSE_CIPHER_SUITES>`_.
+
+Run the following single-line code on the same JVM from the same vendor as
+configured on the coordinator to determine that JVM's default cipher list.
+
+.. code-block:: java
+
+  jrunscript -e "java.util.Arrays.asList(javax.net.ssl.SSLServerSocketFactory.getDefault().getSupportedCipherSuites()).stream().forEach(println)"
```

Comment:
This doesn't work on Java 17 (with Zulu JDK on macOS):
```
script engine for language js can not be found
```
This should work on both:
```shell
echo "java.util.Arrays.asList(((javax.net.ssl.SSLServerSocketFactory) javax.net.ssl.SSLServerSocketFactory.getDefault()).getSupportedCipherSuites()).stream().forEach(System.out::println)" | jshell -

Reply:
OK. Fixing to use the better script. Thanks!